### ISSUE TITLE
Impostos da Empresa poderiam ser mais genéricos / divididos entre compra e vendas.

### ISSUE THREAD

**Issue by mileo** [OCA/l10n-brazil#3259]
Hoje para uma empresa de serviços não é possível configurar os impostos de serviços e suas retenções no cadastro da empresa, como por exemplo:

![image](https://github.com/user-attachments/assets/11a519e7-f2cb-44ef-bf43-7d253ef1a8fc)

Pois quando a empresa vai efetuar uma compra ou outra operação com documento fiscal, todas as retenções são aplicadas.

Então no fim das contas somente o pis e cofins (sem suas retenções) podem ser utilizados no cadastro de uma empresa.

**Comment by antoniospneto** [comment]
mas a solução não seria configurar isso na operação fiscal?

**Comment by mileo** [comment]
> mas a solução não seria configurar isso na operação fiscal?

Sim, mas se for só lá, os campos nem deveriam existir na empresa....

**Comment by antoniospneto** [comment]
ah você fala desses campos então:

![image](https://github.com/user-attachments/assets/b037e7a3-5f88-481a-98ce-b7dfa330b867)

Eu a principio não vejo necessidade também, se ninguem se importar talvez possa ser removido

**Comment by douglascstd** [comment]
Para os campo de imposto padrão fixado na empresa, poderia sim ter um atributo para indicar se esse imposto padrão é de entrada ou saída, ou então pela operação. Isso ajudaria a aplicação destes impostos de maneira correta. Como disse o @mileo, se esses impostos são indicados, no comportamento atual, esses impostos entram em todas as operações. Essa proposta de alteração deve impactar todos os cálculos para que seja verificado mais esse atributo a este imposto e empresa.

Em contra partida, o uso das Operações e linhas de operações fiscais resolvem essa questão de imposto padrão para vendas de serviço.

Isso posto, vejo que não tem um problema específico, mas uma questão de conceito a ser utilizado na configuração do imposto. Poderia manter o imposto padrão na empresa para impostos gerais (como mencionado PIS e Cofins) e os particulares pela operação executada, na **linha de operação fiscal**
